### ISSUE TITLE
About data split.

### ISSUE THREAD

**Issue by liujing1003** [TropComplique/image-classification-caltech-256#2]
Hello, I want to do some research on image classification tasks using Caltech256 dataset recently. Have you participated in the image classification competition of Kaggle on this dataset? I wonder if you could tell me the official split way of the training set and test set? I see you have used 20 images per class for the test set, right? Is it the official split method? Look forward to your reply, thank you.
